apple/internal/ios_rules.bzl

```diff
@@ -899,6 +903,183 @@ def _ios_extension_impl(ctx):
         ),
     ] + processor_result.providers
 
+def _ios_dynamic_framework_impl(ctx):
+    """Experimental implementation of ios_dynamic_framework."""
+    deps_list = [deps for deps in ctx.attr.deps]
+    binary_target = deps_list.pop()
+    extra_linkopts = []
+    if ctx.attr.extension_safe:
+        extra_linkopts.append("-fapplication-extension")
+
+    link_result = linking_support.register_linking_action(
+        ctx,
+        extra_linkopts = extra_linkopts,
+    )
+    binary_artifact = link_result.binary_provider.binary
+    debug_outputs_provider = link_result.debug_outputs_provider
+
+    actions = ctx.actions
+    bin_root_path = ctx.bin_dir.path
+    bundle_id = ctx.attr.bundle_id
+    bundle_name, bundle_extension = bundling_support.bundle_full_name_from_rule_ctx(ctx)
+    executable_name = bundling_support.executable_name(ctx)
+    entitlements = entitlements_support.entitlements(
```

Comment:
I know this is copied and pasted from other very similar implementations, but can `entitlements` be moved above `executable_name` to make this alphabetical again?